src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -38,6 +38,12 @@ export type OnMoveTextNodeCallbackType = (
 ) => void;
 
 export type OnSelectNodeCallbackType = (equipmentId: string, nodeId: string) => void;
+export type OnToggleHoverCallbackType = (
```

Comment:
You noted in a previous comment that you would rename it on SingleLineDiagramViewer side also. Could you do it?
Hence, to avoid clashing with the similar callback from the other viewer, we should add "Nad" keyword inside the type (in the type and in the demo only, the variable names inside the viewers shouldn't clash)

Reply:
done